### PR TITLE
Easy talk approach

### PR DESCRIPTION
**Pull request by sergiobayona**

### CODE REVIEW

File: instructor-rb.gemspec [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require_relative 'lib/instructor/version'
+
+Gem::Specification.new do |spec|
+  spec.name = 'instructor-rb'
+  spec.version = Instructor::VERSION
+  spec.authors = ['Jason Liu', 'Sergio Bayona']
```
Comment (jxnl) [inline instructor-rb.gemspec]: you should go first :)

**Comment by jxnl** [comment]
is this good to merge?

**Comment by sergiobayona** [comment]
> is this good to merge?

I think so. Clearly it's missing many features but at least it's a starting point. I would like to continue adding things like async/parallel requests, async retry, etc...

**Comment by sergiobayona** [comment]
If it's okay. I can run a release which involves publishing to Rubygems.org. Unless you prefer to have a more robust version.

**Comment by jxnl** [comment]
lets test it!

**Comment by jxnl** [comment]
lets merge and then try to make sure docs are accurate

**Comment by jake** [comment]
just wanted to say thanks for making this ruby port! looking forward to trying it out :D

**Pull request merged**